app/Models/Dashboard.php

```diff
@@ -72,9 +72,9 @@ public function widgets()
      * @param $user_id
      * @return Builder|static
      */
-    public function scopeAllAvailable(Builder $query, $user_id)
```

Comment:
Reason for this change was thinking we should in general prefer passing models over ids.